assets/dev/js/frontend/hello-frontend.js

```diff
@@ -0,0 +1,33 @@
+( function( $ ) {
+    $( document ).on( 'click', '.site-navigation-toggle', ( event ) => {
+        const $menuToggle = $( event.currentTarget ),
+              $menuToggleHolder = $menuToggle.parent( '.site-navigation-toggle-holder' ),
+              $dropdownMenu = $menuToggleHolder.siblings( '.site-navigation-dropdown' ),
+              isDropdownVisible = ! $menuToggleHolder.hasClass( 'elementor-active' );
+
+        $menuToggle.attr( 'aria-expanded', isDropdownVisible );
+        $dropdownMenu.attr( 'aria-hidden', ! isDropdownVisible );
+        $menuToggleHolder.toggleClass( 'elementor-active', isDropdownVisible );
+
+        // Always close all sub active items.
+        $dropdownMenu.find( '.elementor-active' ).removeClass( 'elementor-active' );
+    } );
+
+    $( document ).on( 'click', '.site-navigation-dropdown .menu-item-has-children > a', ( event ) => {
+        const $anchor = $( event.currentTarget ),
+              $parentLi = $anchor.parent( 'li' ),
+              isSubmenuVisible = $parentLi.hasClass( 'elementor-active' );
+
+        if ( ! isSubmenuVisible ) {
+            $parentLi.addClass( 'elementor-active' );
+        } else {
+            $parentLi.removeClass( 'elementor-active' );
+        }
+    } );
+
+    $( window ).on( 'resize', () => {
```

Comment:
Poor performance

Reply:
This requires some throttling, or some way to improve its performance, since it is an expensive action that happens on every resize. When a user manually changes their viewport size, e.g. by dragging - this has the potential to run dozens to hundreds of times a second.

Reply:
What would you recommend instead of this method @KingYes